tests/BuilderTest.php

```diff
@@ -792,6 +792,12 @@ public function test_limit()
         $this->assertEquals('SELECT * FROM `table` LIMIT 2, 1', $builder->toSql());
     }
 
+    public function test_totals()
+    {
+        $builder = $this->getBuilder()->from('table');
```

Comment:
Unfortunately this test failed:

![image](https://user-images.githubusercontent.com/20535375/66661543-dca6c800-ec4f-11e9-9ec7-af7f510da974.png)

Did you forget to add `withTotals()` ?